scripts/cliff.toml

```diff
@@ -38,16 +38,27 @@ footer = """
 [git]
 # parse the commits based on https://www.conventionalcommits.org
 conventional_commits = true
-# filter out the commits that are not conventional
-filter_unconventional = true
+# do not filter out the commits that are not conventional
+filter_unconventional = false
+# do not filter out the commits that are not matched by commit parsers
+filter_commits = false
 # process each line of a commit as an individual commit
 split_commits = false
 # regex for preprocessing the commit messages
 commit_preprocessors = [
 #    { pattern = '\((\w+\s)?#([0-9]+)\)', replace = "([#${2}](https://github.com/neovim/neovim/issues/${2}))"},
+    { pattern = '^\[Backport release.*\]\s*', replace = '' },
+    { pattern = '[\s\r\n]+$', replace = '' },  # strip trailing empty lines
 ]
 # regex for parsing and grouping commits
 commit_parsers = [
+    # Exclude "merge commits"
+    { message = '^Merge pull request #[0-9]+ from .*[\n\s]*', skip = true },
+    { message = '^Merge #[0-9]+', skip = true },
```

Comment:
would it be more reliable to filter merge commits like this: https://git-cliff.org/docs/tips-and-tricks#filter-merge-commits

i.e. change line 17 to : 

    {% for group, commits in commits | group_by(attribute="group") | filter(attribute="merge_commit", value=false) %}